node/src/components/consensus/highway_core/active_validator.rs

```diff
@@ -292,24 +292,6 @@ impl<C: Context> ActiveValidator<C> {
         true
     }
 
-    /// Returns the panorama of the confirmation for the leader unit `vhash`.
-    fn confirmation_panorama(&self, vhash: &C::Hash, state: &State<C>) -> Panorama<C> {
```

Comment:
I've moved it to `State` so that we can unit test it only with `State` and without having to interact with `ActiveValidator`. Also, `citable_panorama` is already part of `State` so it's not terrible either.